docs/JSON-RPC-API-using-jmwalletd.md

```diff
@@ -0,0 +1,118 @@
+## JSON-RPC API for Joinmarket using jmwalletd.py
+
+### Introduction - how to start the server
+
+After installing Joinmarket as per the [INSTALL GUIDE](INSTALL.md), navigate to the `scripts/` directory as usual and start the server with:
+
+```
+(jmvenv) $python jmwalletd.py
+```
+
+which with defaults will start serving the RPC over HTTP on port 28183.
+
+This HTTP server does *NOT* currently support multiple sessions; it is intended as a manager/daemon for all the Joinmarket services for a single user.
+
+#### Rules about making requests
+
+Currently authentication is done by providing a cookie on first request, which must then be reused to keep the same session. The cookie is sent in an HTTP header with name `b'JMCookie'`. This is fine for an early testing stage, but will be improved/reworked, and that will be documented here.
```

Comment:
I think it'd be good to rewrite this now, so if anyone's testing it or reviewing it they know the correct session management technique (this was just a placeholder at the time it was written).